routes/sessionRoutes.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+// const passport = require("passport");
```

Comment:
passport is undefined. need to uncomment this line
```suggestion
const passport = require("passport");
```